client/notebooks/Demo_Client_Notebook_Smartnoise-SQL.ipynb

```diff
@@ -165,633 +164,456 @@
-    "Therefore, she calls the fonction `get_initial_budget`."
+    "# On the remote server dummy dataframe\n",
+    "dummy_res = client.smartnoise_sql_query(\n",
+    "    query = QUERY,  \n",
+    "    epsilon = EPSILON,\n",
+    "    delta = DELTA,\n",
+    "    mechanisms = {\"count\": \"gaussian\", \"sum_float\": \"laplace\"},\n",
+    "    dummy = True,\n",
+    ")"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
-   "id": "61a467f3",
+   "execution_count": 15,
+   "id": "46e064f0-f1e2-49af-8f14-fde44f981813",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'initial_epsilon': 10.0, 'initial_delta': 0.005}"
+       "'Average bill length on dummy: 42.39mm.'"
       ]
      },
-     "execution_count": 12,
+     "execution_count": 15,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "client.get_initial_budget()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "bc8f7a74",
-   "metadata": {},
-   "source": [
-    "She sees that she has 10.0 epsilon and 0.0004 epsilon at her disposal.\n",
-    "\n",
-    "Then she checks her total spent budget `get_total_spent_budget`. As she only did queries on metadata on dummy dataframes, this should still be 0."
+    "avg_bl_dummy = np.round(dummy_res['query_response'][\"avg_bill_length_mm\"][0], 2)\n",
+    "f\"Average bill length on dummy: {avg_bl_dummy}mm.\""
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
-   "id": "afd22f84",
+   "execution_count": 16,
+   "id": "7e20014d-ad82-4a2d-88d9-ec981150e7db",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'total_spent_epsilon': 2.714285714286655, 'total_spent_delta': 0.0}"
+       "{'epsilon_cost': 1.0, 'delta_cost': 1.4999949999983109e-05}"
       ]
      },
-     "execution_count": 13,
+     "execution_count": 16,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "client.get_total_spent_budget()"
+    "cost = client.estimate_smartnoise_sql_cost(\n",
+    "    query = QUERY, \n",
+    "    epsilon = EPSILON, \n",
+    "    delta = DELTA,\n",
+    "    mechanisms = {\"count\": \"gaussian\", \"sum_float\": \"laplace\"}\n",
+    ")\n",
+    "cost"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "05daf5a4",
+   "id": "e5379edf",
    "metadata": {},
    "source": [
-    "It will also be useful to know what the remaining budget is. Therefore, she calls the function `get_remaining_budget`. It just substarcts the total spent budget from the initial budget."
+    "### Query real dataset\n",
+    "Dr. Antartica is ready to query on the real dataset and get a differentially private response of the average bill length. By default, the flag `dummy` is False so setting it is optional. She uses the values of `epsilon` and `delta` that she selected just before.\n",
```

Comment:
The perform her query on the real dataset OR to query the real dataset.
A dp response for the average bill length.
The "dummy" flag is False by default, so setting it is optional.